tests/method-chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -126,10 +126,12 @@ exports[`comment.js 1`] = `
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 function f() {
-  return observableFromSubscribeFunction()
-    // Debounce manually rather than using editor.onDidStopChanging so that the debounce time is
-    // configurable.
-    .debounceTime(debounceInterval);
+  return (
+    observableFromSubscribeFunction()
+      // Debounce manually rather than using editor.onDidStopChanging so that the debounce time is
```

Comment:
this is a bummer; see https://github.com/prettier/prettier/pull/955/files#r105058093 for explanation.